src/referee/referee/publisher_referee.py

```diff
@@ -0,0 +1,35 @@
+import rclpy
+from rclpy.node import Node
+from google.protobuf import text_format
+
+
+class MinimalPublisher(Node):
+
+    def __init__(self):
+        super().__init__('referee_publisher')
+
+        self.declare_parameter('ip', '224.5.23.1')
+        self.declare_parameter('port', 10003)
+        self.publisher_ = self.create_publisher(String, 'topic', 10)
```

Comment:
This is sending a String message, instead, it needs to be a custom message format in System Interfaces module to process game controller protobufs